### PR TITLE
[DOCFIX] Corrected link from Architecture.md doc on both languages

### PR DESCRIPTION
**Pull request by caadecarvalho**
The link referred to Alluxio Faut Tolerant Mode on EC2 machines, so we fixed the link to Alluxio Fault Tolerant Mode site.

### CODE REVIEW

**Comment by caadecarvalho** [comment]
Jenkins, test this please.

File: docs/en/Architecture.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ client portion of Alluxio.
 ### Master
 
 Alluxio may be deployed in one of two master modes, [single master](Running-Alluxio-Locally.html) or
-[fault tolerant mode](Running-Alluxio-Fault-Tolerant-on-EC2.html). The master is primarily
+[fault tolerant mode](Running-Alluxio-Fault.html). The master is primarily
```
Comment (aaudiber) [inline docs/en/Architecture.md]: Fault-Tolerant

Comment (caadecarvalho) [inline docs/en/Architecture.md]: @aaudiber , I have no idea why it is just like this as in my notebook, it is correct:

`MacBook-Air:docs caadecarvalho$ grep Fault en/Architecture.md`
`[fault tolerant mode](Running-Alluxio-Fault-Tolerant.html). The master is primarily`
`MacBook-Air:docs caadecarvalho$ git status en/Architecture.md` 
`On branch smallfix`
`nothing to commit, working directory clean`

Anyway, I'll fix it through the web.

**Comment by caadecarvalho** [comment]
Jenkins, test this please.

**Comment by aaudiber** [comment]
LGTM, thanks for the improvement

**Pull request merged**